en/docs/reference/connectors/amazonsqs-connector/amazonsqs-connector-reference.md

```diff
@@ -0,0 +1,1058 @@
+
+??? note "init"  
+    The init operation is used to initialize the connection to Amazon SQS.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>name</td>
+            <td>Unique name to identify the connection by.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>region</td>
+            <td>The regional endpoint to make your requests (e.g., us-east-1).</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>secretAccessKey</td>
+            <td>TA secret access key associated with your AWS access key.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>accessKeyId</td>
+            <td>The access key ID associated with your AWS account.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionMaxIdleTime</td>
+            <td>The maximum amount of time(in seconds) that a connection should be allowed to remain open while idle.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionTimeout</td>
+            <td>The amount of time(in seconds) to wait when initially establishing a connection before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionTimeToLive</td>
+            <td>The maximum amount of time(in seconds) that a connection should be allowed to remain open, regardless of usage frequency.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>connectionAcquisitionTimeout</td>
+            <td>The amount of time(in seconds) to wait when acquiring a connection from the pool before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>socketTimeout</td>
+            <td>The amount of time(in seconds) to wait for data to be transferred over an established, open connection before the connection is timed out.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallTimeout</td>
+            <td>The amount of time(in seconds) to allow the client to complete the execution of an API call.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>apiCallAttemptTimeout</td>
+            <td>The amount of time(in seconds) to wait for the http request to complete before giving up and timing out.</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    > **Note**: You can either pass credentials within the `init` configuration or set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as environment variables. The AWS SDK uses provider chains to look for AWS credentials in system/user environment variables.
+        
+    To set these environment variables on Linux, macOS, or Unix, use export :
```

Comment:
Shall we write the following as [content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/?h=tabs) for better clarity?